doc/snapper-configs.xml.in

```diff
@@ -266,6 +266,30 @@
     </variablelist>
   </refsect1>
 
+  <refsect1 id='types'>
+    <title>TYPES</title>
+    <para>Some of the types used for the variables deserve an explanation.</para>
+    <variablelist>
+
+      <varlistentry>
+	<term>range</term>
+	<listitem>
+	  <para>A range of numbers given as
+	  &lt;max-value&gt;-&lt;min-value&gt;, e.g. &quot;10-20&quot;.</para>
```

Comment:
Reversed? min-value, max-value
I'm not sure of the rendering, but would it be better to use the "replaceable" tag instead of angle brackets?